src/actions/admin.ts

```diff
@@ -22,129 +21,117 @@ export function AdminActions<ActionsBase extends Constructor<TinlakeParams>>(Bas
+    setMaximumJuniorRatio = async (ratio: string) => {
       // Source: https://github.com/ethereum/web3.js/issues/2256#issuecomment-462730550
+      const minSeniorRatio = (10 ** 27 - Number(ratio)).toString()
       return this.pending(
-        this.contract('PILE').file(web3.fromAscii('rate').padEnd(66, '0'), rateGroup, ratePerSecond, this.overrides)
+        this.contract('ASSESSOR').file(web3.fromAscii('minSeniorRatio').padEnd(66, '0'), minSeniorRatio, this.overrides)
       )
     }
 
-    changeRate = async (loan: string, ratePerSecond: string) => {
-      const rateGroup = getRateGroup(ratePerSecond)
-      return this.pending(this.contract('PILE').changeRate(loan, rateGroup, this.overrides))
+    setMaximumReserve = async (value: string) => {
+      // Source: https://github.com/ethereum/web3.js/issues/2256#issuecomment-462730550
+      return this.pending(
+        this.contract('ASSESSOR').file(web3.fromAscii('maxReserve').padEnd(66, '0'), value, this.overrides)
+      )
     }
 
-    setRate = async (loan: string, ratePerSecond: string) => {
-      const rateGroup = getRateGroup(ratePerSecond)
-      return this.pending(this.contract('PILE').setRate(loan, rateGroup, this.overrides))
+    setSeniorTrancheInterest = async (value: string) => {
+      // Source: https://github.com/ethereum/web3.js/issues/2256#issuecomment-462730550
+      return this.pending(
+        this.contract('ASSESSOR').file(web3.fromAscii('seniorInterestRate').padEnd(66, '0'), value, this.overrides)
+      )
     }
 
-    // ------------ admin functions lender-site -------------
-    setMinimumJuniorRatio = async (ratio: string) => {
+    setDiscountRate = async (rate: string) => {
       // Source: https://github.com/ethereum/web3.js/issues/2256#issuecomment-462730550
       return this.pending(
-        this.contract('ASSESSOR').file(web3.fromAscii('minJuniorRatio').padEnd(66, '0'), ratio, this.overrides)
+        this.contract('FEED').file(web3.fromAscii('discountRate').padEnd(66, '0'), rate, this.overrides)
       )
     }
 
-    approveAllowanceJunior = async (user: string, maxCurrency: string, maxToken: string) => {
-      return this.pending(this.contract('JUNIOR_OPERATOR').approve(user, maxCurrency, maxToken, this.overrides))
+    // TODO: setMaturityDate (maybe not needed for MVP)
+    setMaturityDate = async (ratio: string) => {
     }
 
-    approveAllowanceSenior = async (user: string, maxCurrency: string, maxToken: string) => {
-      if (this.getOperatorType('senior') === 'PROPORTIONAL_OPERATOR') {
-        return this.pending(this.contract('SENIOR_OPERATOR').approve(user, maxCurrency, this.overrides))
-      } 
-        return this.pending(this.contract('SENIOR_OPERATOR').approve(user, maxCurrency, maxToken, this.overrides))
-      
-      
+    updateJuniorMemberList = async (user: string, validUntil: number) => {
+      return this.pending(this.contract('JUNIOR_MEMBERLIST')['updateMember(address,uint)'](user, validUntil, this.overrides))
```

Comment:
This type of object access for the functions is only needed if there are multiple functions in the contract with the same name (function overloads). I think that's not the case here, so it can be simplified.
```suggestion
      return this.pending(this.contract('JUNIOR_MEMBERLIST').updateMember(user, validUntil, this.overrides))
```